pyembed/Cargo.toml

```diff
@@ -47,6 +48,7 @@ rusty-fork = "0.3"
 [features]
 default = ["build-mode-default"]
 jemalloc = ["jemalloc-sys"]
+gmimalloc = ["mimalloc"]
```

Comment:
Why the _g_?

Reply:
Rust would not allow everything to compile if the feature name was the same as the actual crate, therefore I added a g to name in reference to it being set "globally" or as the global rust allocator.